packages/client3/components/design-system/input/input.tsx

```diff
@@ -47,6 +47,7 @@ interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
    */
   labelDecoration?: ReactNode;
   textSize?: "sm" | "md" | "lg" | "xl";
+  money?: boolean;
```

Comment:
Very minor thing Would prefer a name that suggests the intent of the prop. Something like isMoneyInput. thoughts?

Reply:
Yeap sure. Good to know